src/mono_font/mono_text_style.rs

```diff
@@ -530,6 +524,11 @@ mod tests {
         ..FONT_6X9
     };
 
+    const _CAN_BUILD_CONST: MonoTextStyle<'static, BinaryColor> = MonoTextStyleBuilder::new()
```

Comment:
I'd probably move that to a test fn

Reply:
It would be the same thing, just with more lines, and it would also create a test that asserts nothing, and does nothing except print that it passes. Would it really be better?

Reply:
personal preference: either test fn or remove entirely - it's just a showcase, right? Either way no strong feelings